Portal_Perfomance_Employes/Controllers/EmployeController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Portal_Perfomance_Employes.Data;
+using Portal_Perfomance_Employes.Models;
+
+namespace Portal_Perfomance_Employes.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmployeController : ControllerBase
+    {
+        private EmployesDB _employesDB = new EmployesDB();
+
+        public EmployeController()
+        {
+            _employesDB.PopularEmployesDB();
+        }
+
+
+        [HttpGet]
+        public async Task<IEnumerable<Employe>> Get() => _employesDB.employes.ToList();
+        
+        [HttpGet("{id}")]
+        public IActionResult GetbyId(int id)
+        {
+
+            Employe emp = _employesDB.employes.Find(x => x.Id == id);
+
+            return emp == null ? NotFound() : Ok(emp) ;
```

Comment:
I like the usage of `Ok()` here- maybe we should extend it to all other request types?

Reply:
I used the OK here because I don't have a list to return like other methods and the return of this method does not accept my object directly, so Ok return status 200 and accept as parameter the content of result to return on the body.

Reply:
You can pass a list to an `Ok()` call and it will work fine